mlir/unittests/programs/qc_programs.cpp

```
@@ -126,8 +126,8 @@ void multipleClassicalRegistersAndMeasurements(QCProgramBuilder& b) {
 }
 
 void resetQubitWithoutOp(QCProgramBuilder& b) {
```

Cppcheck (2.20.0)

[style] The function 'resetQubitWithoutOp' is never used.

(unusedFunction)